src/internal/neynar/user/neynarUserFunctions.ts

```diff
@@ -0,0 +1,78 @@
+import { FetchError } from '../exceptions/FetchError';
+
+export interface NeynarUserModel {
+  fid: number;
+  custody_address: string;
+  username: string;
+  display_name: string;
+  pfp_url: string;
+  profile: {
+    bio: {
+      text: string;
+    };
+  };
+  follower_count: number;
+  verifications: string[];
+}
+export interface NeynarBulkUserLookupModel {
+  users: NeynarUserModel[];
+}
+
+export class NeynarUserFunction {
```

Comment:
QQ: would we be better off with [the official SDK](https://www.npmjs.com/package/@neynar/nodejs-sdk)? Implementing our own client on top of their HTTP api is more work than we might want to do over the long run.

One con to using the SDK would be that the bundlesize is a bit high: https://bundlephobia.com/package/@neynar/nodejs-sdk@1.7.0 though because this would be running server-side it's not as big a deal cc @Zizzamia

Reply:
I would be open to it.. looks like most of the library is auto-generated.. I wanted to keep our imports lightweight, but it is a great question on future support / extensibility.